### PR TITLE
adding mmv2-titanic

### PR DESCRIPTION
**Pull request by pallavipannu**

### CODE REVIEW

File: titanic/modelmonitor.ipynb [hunk 1]
```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "#### MM-SDK"
+    "#### MMV2-SDK"
```
Comment (songole) [inline titanic/modelmonitor.ipynb]: Do we need this heading? Please remove it. Going fwd. APIs won't change much

File: titanic/performance_soft_thresholds.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"mse": {"baseline":0.2,"soft":0.3}}
```
Comment (songole) [inline titanic/performance_soft_thresholds.json]: Lets revise the SDK when soft/hard tomorrow and update both the examples. Upload thresholds for all metrics and not just one.

File: titanic/README.ui.md [hunk 3]
```diff
@@ -41,28 +41,31 @@ If it is external, click on Deployments in the left tab and import a deployment
 4. Click Next and save.
 
 ### 4. Performance Monitoring
-1. Check Enabled option and provide frequency as 5 minutes and upload soft thresholds from [link]([link](https://raw.githubusercontent.com/oneconvergence/dkube-examples/monitoring/insurance/performance_soft_thresholds.json)
+1. Check Enabled option and provide frequency as 5 minutes and upload soft thresholds from [link]([link](https://raw.githubusercontent.com/oneconvergence/dkube-examples/monitoring/titanic/performance_soft_thresholds.json)
```
Comment (songole) [inline titanic/README.ui.md:44]: The file name should just be thresholds. Can you fix it in the next PR?

File: titanic/modelmonitor.ipynb [hunk 4]
```diff
@@ -254,7 +247,7 @@
    "outputs": [],
    "source": [
     "mm=DkubeModelmonitor(name=MONITOR_NAME)\n",
-    "mm.update_performance_monitoring_details(enabled=True,source_type=\"labelled_data\",frequency=5,soft_thresholds={\"mse\":{\"baseline\":0.2,\"soft\":0.3}})"
+    "mm.update_performance_monitoring_details(enabled=True,source_type=\"labelled_data\",frequency=5,soft_thresholds={\"accuracy\":{\"hard\":0.890476152,\"soft\":0.816269806},\"precision\":{\"hard\":0.8795999999999999,\"soft\":0.8063},\"f1_score\": {\"hard\":0.67929912,\"soft\":0.62269086}})"
```
Comment (songole) [inline titanic/modelmonitor.ipynb:250]: parameter name should also be just thresholds
@pallavi-pannu-oc @gaurav-oc 
Can you fix it in the next PR?

Comment (gaurav-oc) [reply titanic/modelmonitor.ipynb:250]: in api its thresholds, in sdk its need to be correceted

**Pull request merged**